scipy/_lib/doccer.py

```diff
@@ -3,8 +3,11 @@
 
 import sys
 
-__all__ = ['docformat', 'inherit_docstring_from', 'indentcount_lines',
-           'filldoc', 'unindent_dict', 'unindent_string', 'doc_replace']
+__all__ = [
+    'docformat', 'inherit_docstring_from', 'indentcount_lines',
+    'filldoc', 'unindent_dict', 'unindent_string', 'extend_notes_in_docstring',
+    'replace_notes_in_docstring', 'doc_replace'
+]
```

Comment:
Was this necessary? It looks like updating the wrong `__all__` to me - this one was fine, and the one in `doccer.py` was to be updated to match this one I think.

Reply:
They should both be the same, right? And the `__all__` here was missing a few functions?

Reply:
@rgommers

Reply:
@h-vetinari Any thoughts on this?

Reply:
Yes indeed, this is private anyway and `__all__` is meant to include these functions since they're used in `scipy.stats`.